Assets/HoloToolkit/UX/Scripts/Buttons/ButtonIconProfile.cs

```diff
@@ -0,0 +1,73 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.Buttons
+{
+    /// <summary>
+    /// The base class for button icon profiles
+    /// </summary>
+    public abstract class ButtonIconProfile : ButtonProfile
+    {
+        /// <summary>
+        /// The icon returned when a requested icon is not found
+        /// </summary>
+        public Texture2D _IconNotFound_;
```

Comment:
what's with the underscore prefix and suffix?

Reply:
not sure, should be removed.